### PR TITLE
Doc: Minor nits in documentation

### PR DESCRIPTION
**Pull request by sugetha24**
## Change Summary
Fixed a few nits in documentation to match changes in `schema documentation`

## Related Issue(s)

Fixes #<ISSUE ID>

## How to test
`make webdoc`
`mkdocs serve`

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by JulioPDX — APPROVED** [review]
LGTM, thank you for the quick update!

File: ansible_collections/arista/cvp/docs/_build/cv_container_v3.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ The following options may be specified for this module:
     <td>loose</td>
     <td><ul><li>loose</li><li>strict</li></ul></td>
     <td>
-        <div>Set how configlets are attached/detached on container. If set to strict all configlets not listed in your vars are detached.</div>
+        <div>Set how configlets are attached/detached to containers. If set to strict all configlets not listed in your vars will be detached.</div>
```
Comment (noredistribution) [inline ansible_collections/arista/cvp/docs/_build/cv_container_v3.rst:51]: Should be `to/from` and I guess we should have one sentence

File: ansible_collections/arista/cvp/docs/_build/cv_change_control_v3.rst [hunk 2]
```diff
@@ -69,7 +69,18 @@ The following options may be specified for this module:
     <td></td>
     <td></td>
     <td>
-        <div>The name of the change control. If not provided, one will be generated.</div>
+        <div>The name of the change control, If not provided, one will be generated automatically.</div>
```
Comment (noredistribution) [inline ansible_collections/arista/cvp/docs/_build/cv_change_control_v3.rst:72]: @JulioPDX how's this sentenceless format exactly? For new thoughts, do we capitalize the first letter or not? Like in this case should it be `If` or `if`?

File: ansible_collections/arista/cvp/docs/modules/cv_device_v3.rst.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ The following options may be specified for this module:
 <td>loose</td>
 <td><ul><li>loose</li><li>strict</li></ul></td>
 <td>
-    <div>Set how configlets are attached/detached on device. If set to strict all configlets and image bundles not listed in your vars are detached.</div>
+    <div>Set how configlets are attached/detached on device. If set to strict, all configlets and image bundles not listed in your vars are detached.</div>
```
Comment (noredistribution) [inline ansible_collections/arista/cvp/docs/modules/cv_device_v3.rst.md:39]: `to/from a device`

File: ansible_collections/arista/cvp/docs/_build/cv_container_v3.rst (hunk 1 above)
Comment (sugetha24) [reply ansible_collections/arista/cvp/docs/_build/cv_container_v3.rst:51]: This is just to match what's already there in schema documentation. We have to change both later.

File: ansible_collections/arista/cvp/docs/modules/cv_device_v3.rst.md (hunk 3 above)
Comment (sugetha24) [reply ansible_collections/arista/cvp/docs/modules/cv_device_v3.rst.md:39]: same comment as above

**Review by noredistribution — APPROVED** [review]
we decided to take the minor nits in the next rel. LGTM +1

**Pull request merged**